reactioncommerce/reaction
# refactor: npm discount codes

Resolves #6123 
Impact: **minor**
Type: **feature|refactor**

## Issue
Move Discount Codes plugin to npm.

## Solution
Created npm package for discount code plugin
https://www.npmjs.com/package/@reactioncommerce/plugin-discount-codes

## Testing
1. Run reaction in dev mode with this branch
2. Create at least one discount code via Reaction Admin
3. Using GraphQL playground and a valid authorization code with the admin role run a discount codes query such as this:
```
{
  discountCodes(shopId: "Your Shop Id") {
    nodes {
      _id
      discount
      transactions {
        cartId
      }
      discountMethod
      description
    }
  }
}
```

4. Ensure you see discount code data returned
